CHANGES/8481.bugfix.rst

```diff
@@ -1,2 +1,4 @@
 Fixed the incorrect rejection of ``ws://`` and ``wss://`` urls
+when creating requests. This was a regression introduced in
+#6722.
```

Comment:
This is the role that our sphinx config has built in. Also, let's drop the trailing period since the byline is attached to the paragraph and is not on a separate line: 
```suggestion
:pr:`6722`
```